packages/terra-notification-dialog/tests/wdio/notification-dialog-spec.js

```diff
@@ -120,9 +120,14 @@ Terra.describeViewports('NotificationDialog with additional focus trap sources w
 
     Terra.validates.element('Close popup', { selector: '[class*="abstract-modal"]' });
 
+    // Skip color contrast check for elements behind the notification dialog's overlay
+    const ignoredA11y = {
+      'color-contrast': { enabled: false },
```

Comment:
Lets scope this color-contrast check to the notification dialog itself, otherwise we drop this check which is commonly seen as an accessibility issue:
```
const scopeA11y = {
    'color-contract': {
          selector: '[class*="abstract-modal]'
    }
}

```

ref: https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#parameters-1

Reply:
good call. updated

Reply:
actually overlay is the component in question - updated the selector to reference that